### ISSUE TITLE
Tests failing with ParamTools 0.16.0

### ISSUE THREAD

**Issue by jdebacker** [PSLmodels/OG-USA#673]
Tests are failing with ParamTools 0.16.0.  Model seems to run fine, but there is an incompatibility with cached `Specifications` objects used in tests (created with an older version of ParamTools) and the new version.

We'll need to figure out how to have old object conform with new ParamTools, or save new `Specifications` objects for testing.

**Comment by hdoupe** [comment]
@jdebacker Sorry about this. I tested against Tax-Calculator, CCC, Tax-Brain, and Tax-Cruncher. For the sake of my computer, I skipped OG-USA. What are the error messages?

**Comment by jdebacker** [comment]
@hdoupe I wouldn't expect you to test against OG-USA!  This will happen, but I appreciate you chiming in here as maybe you can suggest a quick fix.

Here's the traceback:
```
____________ ERROR collecting ogusa/tests/test_parameter_tables.py _____________
66
ImportError while importing test module '/home/runner/work/OG-USA/OG-USA/ogusa/tests/test_parameter_tables.py'.
67
Hint: make sure your test modules/packages have valid Python names.
68
Traceback:
69
/usr/share/miniconda/envs/ogusa-dev/lib/python3.7/importlib/__init__.py:127: in import_module
70
    return _bootstrap._gcd_import(name[level:], package, level)
71
ogusa/tests/test_parameter_tables.py:15: in <module>
72
    os.path.join(CUR_PATH, 'test_io_data', 'model_params_baseline.pkl'))
73
ogusa/utils.py:364: in safe_read_pickle
74
    obj = pickle.load(f, encoding='latin1')
75
E   ModuleNotFoundError: No module named 'paramtools.tree'
```

**Comment by hdoupe** [comment]
Oh, the `tree` module was removed in 0.16.0...you may be able to dump the parameter values and load them with the new version of paramtools:

```
conda install "paramtools==0.15.1" -c conda-forge
```

```python
obj = pickle.load(f, encoding='latin1')

with open("model_params_baseline.json", "w") as f:
    f.write(
        json.dumps(obj.specification(use_state=False, meta_data=False, serializable=True))
    )

```

and then:

```
conda install "paramtools==0.16.0" -c conda-forge
```

```python

params = Specifications() 

params.adjust("model_params_baseline.json")

```

However, this will only work if all parameter values are updated using `adjust`. I think there are some places where OG-USA just updates the parameter value directly like this:

https://github.com/PSLmodels/OG-USA/blob/54d5807fcf398fda05bc142c910ff52b8630082d/ogusa/parameters.py#L131-L145

**Comment by jdebacker** [comment]
This issue was resolved in PR #674, which updated the 6 pickle files that contain `Specifications` objects with those created using ParamTools 0.16.0.

Note that there were issues reading pickle files created in Python 3.8 in Python 3.7 for the tests run via GitHub actions.  GH Actions now only tests against Python 3.8 (Travis CI continues to test against 3.7 and 3.8).
